core-services/image-mirroring/openshift/mapping_origin_ci-operator-4.17

```diff
@@ -10,10 +10,7 @@ registry.ci.openshift.org/ci/repo-init:latest quay.io/openshift/ci-public:ci_rep
 registry.ci.openshift.org/ci/ci-operator-configresolver:latest quay.io/openshift/ci-public:ci_ci-operator-configresolver_latest
 registry.ci.openshift.org/ci/python-validation:latest quay.io/openshift/ci-public:ci_python-validation_latest
 registry.ci.openshift.org/ci/cluster-init:latest quay.io/openshift/ci-public:ci_cluster-init_latest
-registry.ci.openshift.org/ci/dashboards-validation:latest quay.io/openshift/ci-public:ci_dashboards-validation_latest
```

Comment:
we need this on. Its used [here](https://github.com/openshift/release/blob/11f36d43c71a38e5778cd05f16869337fb6c6caf/Makefile#L305)